workshops/models.py

```diff
@@ -343,11 +343,6 @@ def is_staff(self):
         """
         return self.is_superuser
 
-    @property
-    def is_admin(self):
-        is_admin = self.groups.filter(name='Admins').exists()
-        return is_admin or self.is_staff or self.is_superuser
-
```

Comment:
`user` can be `None`, `AnonymousUser` or `Person`. You can cover all three cases if you use a function instead of a method.